chia/wallet/wallet_protocol.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List, Optional, Set
+
+from blspy import G1Element
+from typing_extensions import Protocol
+
+from chia.server.ws_connection import WSChiaConnection
+from chia.types.blockchain_format.coin import Coin
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.util.ints import uint8, uint32, uint64, uint128
+from chia.wallet.wallet_coin_record import WalletCoinRecord
+
+if TYPE_CHECKING:
+    from chia.wallet.wallet_state_manager import WalletStateManager
+
+
+class WalletProtocol(Protocol):
+
+    # TODO: it seems like this should return WalletType instead
```

Comment:
Just verifying this is meant to be left in when merged.

Reply:
I kind of meant to leave it in. I would like to explore making that change, but I would need mypy coverage first.